org.kde.kruler.json

```diff
@@ -0,0 +1,28 @@
+{
+    "id": "org.kde.kruler",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kruler",
+    "rename-icon": "kruler",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+
+    "modules": [
+        {
+            "name": "kruler",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                { "type": "git",
+                  "commit": "0077a55ea0cb8acfc5b02bd072d41f458546c5e9",
```

Comment:
You need to use a tarball not a git commit

Reply:
https://github.com/flathub/flathub/pull/2750/commits/443bfbb1d0f2ba5ef85ff9968a9b93981ec94aa7